SU2_CFD/src/fluid/CUserDefinedTCLib.cpp

```diff
@@ -630,13 +630,14 @@ void CUserDefinedTCLib::SetTDStateRhosTTv(vector<su2double>& val_rhos, su2double
 
 vector<su2double>& CUserDefinedTCLib::GetSpeciesCvTraRot(){
 
+  //TODO: Set this in constructor.
   for (iSpecies = 0; iSpecies < nHeavy; iSpecies++)
     Cvtrs[iSpecies] = (3.0/2.0 + RotationModes[iSpecies]/2.0) * Ru/MolarMass[iSpecies];
 
   return Cvtrs;
 }
```

Comment:
question: I understand the suggestion, since it is a constant variable afterall. I wonder if we want to keep it here in case we want to go for more detailed models in the future, where this wouldn't be constant. Again, not a big deal, we can always change back. Just to share thoughts

Reply:
Thats a good point.  I think either option is good.  I can update the comment